DisCatSharp.ApplicationCommands/ApplicationCommandsConfiguration.cs

```diff
@@ -46,5 +55,20 @@ public ApplicationCommandsConfiguration(IServiceProvider provider)
         {
             this.ServiceProvider = provider;
         }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="ApplicationCommandsConfiguration"/>, copying the properties of another configuration.
+        /// </summary>
+        /// <param name="acc">Configuration the properties of which are to be copied.</param>
+        public ApplicationCommandsConfiguration(ApplicationCommandsConfiguration acc)
+        {
+            this.EnableDefaultHelp = acc.EnableDefaultHelp;
+            this.ServiceProvider = acc.ServiceProvider;
+        }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="ApplicationCommandsConfiguration"/>.
+        /// </summary>
+        public ApplicationCommandsConfiguration() { }
```

Comment:
@Lulalaby, believe we were trying to keep the dependency injection pipeline the whole way through right? If so, this default constructor would be problematic

Reply:
yes

Reply:
I would suggest changing line 54 to IServiceProvider provider = null and removing 68-72